### DISCUSSION TITLE
Shortcut to pipeline

### DISCUSSION THREAD

**Discussion by AndreYin** [CynToolkit/cyn#11] [Ideas]
If possible, it would be great to export the solution to either a .bat file or an .exe file, once it's all setup. This way we wouldn't require to open CYN and run the solution.

My ideal workflow would be:

1. Export game on C3
2. Run the CYN solution after the export is done

CYN does have an option to load the c3p and export, which is great, but when I'm testing something I already have C3 open - I wonder if it would be possible to "hack" c3 in such a way (maybe with a chromium plugin?) that it checks when it's done exporting, and runs the specified CYN solution.

Maybe that's totally out of the scope of the project, which is fine btw!

**Comment by Armaldio** [comment]
Hey !

Thanks for your suggestion!

I have two things planned:
- create shortcuts for pipelines: as you said, just create a pipeline, then save it to a shortcut
- as you may know, there is a trigger thing at the start of the pipeline. Currently you only have "Manual". But in the future, you access multiple triggers like:
  - when folder content changes
  - every X hours 
  - etc.

Maybe one of these features can fit your workflow ?

**Comment by AndreYin** [comment]
Aaah, yes!! Wow having it to scan the folder for changes would be perfect!

Em qui., 8 de ago. de 2024, 16:48, Quentin Goinaud ***@***.***>
escreveu:
